src/Wallet.purs

```diff
@@ -174,30 +99,107 @@ dummySign tx@(Transaction { witnessSet: tws@(TransactionWitnessSet ws) }) =
           "ed25519_sig1ynufn5umzl746ekpjtzt2rf58ep0wg6mxpgyezh8vx0e8jpgm3kuu3tgm453wlz4rq5yjtth0fnj0ltxctaue0dgc2hwmysr9jvhjzswt86uk"
     )
 
+-------------------------------------------------------------------------------
+-- Gero backend
+-------------------------------------------------------------------------------
+
+mkGeroWalletAff :: Aff Wallet
```

Comment:
We can factor this out (along with `mkNamiWalletAff`) into a `mkCip30WalletAff` that takes an `Effect (Promise Cip30Connection)` HOF argument, along with the name of the wallet as a `String` to include in the error message. It will make it easier to add more CIP-30-compliant wallets in the future